src/runtime_print.cpp

```
@@ -40,12 +41,7 @@ static int append_vprintf(char *&buf, size_t &len, size_t &cap,
     if (static_cast<size_t>(n) >= remaining) {
         size_t newCap = len + static_cast<size_t>(n) + 1;
```

Clang (14.0.6)

[warning] variable 'newCap' is not initialized

(cppcoreguidelines-init-variables)